common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -463,15 +463,16 @@ def test_multi_line_input(self):
         """
 
         inputs = {
-            "key": "value",
-            "key_2": "value_2"
+            "key": {"id": "value_1"},
+            "key_2": {"id": "value_2"}
         }
         json_input = json.dumps(inputs)
         self.advanced_settings.set('Discussion Topic Mapping', json_input)
-        self.assertEqual(
-            self.advanced_settings.get('Discussion Topic Mapping'),
-            '{\n    "key": "value",\n    "key_2": "value_2"\n}'
-        )
+        advanced_setting_get = self.advanced_settings.get('Discussion Topic Mapping')
+        expected_output = u'{\n    "key": {\n        "id": "value_1"\n    },\n    '\
```

Comment:
This is a very brittle test :(

Reply:
Yes, it is

Reply:
I do not know how necessary it is, I personally would love to change it to something else. It took me forever to debug.

Reply:
change or get rid of it